internal/cmd/mockery.go

```diff
@@ -322,5 +356,16 @@ func (r *RootApp) Run() error {
 		}
 	}
 
+	// The loop above could exit early, so sometimes warnings won't be shown
+	// until other errors are fixed
+	for packageName := range missingMap {
```

Comment:
```suggestion
	for packagePath := range missingMap {
```

Technically, a "package name" is different from a "package path".

Reply:
makes sense, fixed